addons/mail/migrations/12.0.1.0/pre-migration.py

```diff
@@ -14,8 +14,20 @@
 ]
 
 
+def fill_mail_message_add_sign_default(env):
+    """Fast way to fill this new field"""
+    if not openupgrade.column_exists(env.cr, 'mail_message', 'add_sign'):
+        openupgrade.logged_query(
+            env.cr, """
+            ALTER TABLE mail_message
+            ADD COLUMN add_sign boolean
+            DEFAULT TRUE""",
```

Comment:
Doubt I have now is this default will act for new additions. I think you must remove that default after this one.

Reply:
how?

Reply:
https://popsql.com/learn-sql/postgresql/how-to-remove-a-default-value-from-a-column-in-postgresql/

Reply:
thanks, done